connect/README.md

```diff
@@ -2,24 +2,63 @@
 
 This package makes it easy for AD4M apps to connect to a local or remote AD4M executor by handling all the complex things like finding the local executor port, requesting and storing a capability token, creating and recreating an Ad4mClient.
 
-![](screenshots/Screenshot_executor_url.png)
-![](screenshots/Screenshot_security_code.png)
+<div style="text-align: center">
+<img src="screenshots/Screenshot_executor_url.png" width="400"></img>
+</div>
 
-## Usage (from Node / Electron)
+## Installation
 
-Install the package:
 `npm install -s @perspect3vism/ad4m-connect`
 
-Import `ad4mConnect()`:
+## Properties
+
+- `appName(required)`: Name of the application using ad4m-connect.
+- `appDesc(required)`: Description of the application using ad4m-connect.
+- `appDomain(required)`: Domain of the application using ad4m-connect.
+- `capabilities(required)`: Capabilities requested by the application.
+- `appIconPath`: Icon for the app using ad4m-connect.
+- `port`: Port that AD4M is running on.
+- `token`: JWT token if you have one.
+- `url`: The url that we should connect to.
+
+
+## Events
+
+- `authstatechange`: `authenticated` | `unauthenticated` | `locked`
+- `connectionstatechange`: `connecting` | `connected` | `not_connected` | `disconnected` | `error`;
+- `configstatechange`: `token` | `url` | `port`
+
+## In the Browser
 
 ```js
-import { ad4mConnect } from '@perspect3vism/ad4m-connect/dist/electron'
+import Ad4mConnectUI from "@perspect3vism/ad4m-connect/dist/web";
+
+const ui = new Ad4mConnectUI({
+  appName: "ad4m-connect-example",
+  appDesc: "hello",
+  appDomain: "http://localhost:3000",

```

Comment:
What prevents other apps from using the same domain name? Should we not maybe use the origin from the request that is made on the AD4M side? This will work great with websites as you need to own the domain to make a request from it, but unsure how it would work in Electron

Reply:
If somebody wants to fake this info, the website domain won't help since we can't assume anything about the sender (i.e. being a web-browser) unless the user checks the connection between the UI they used to trigger the request and the info the launcher shows..